### ISSUE TITLE
Prevent log found with archived virtuals/own/webcam

### ISSUE THREAD

**Issue by kojoty** [opencaching/opencaching-pl#2142]
(continuation of #17)

+ configuration for types which allow implementation of node-specific politics

**Comment by rapotek** [comment]
Clarification question: does it apply to any log found regardless of logging date or only to founds logged with a date after a cache has been archived?

**Comment by kojoty** [comment]
This is optional feature which allow per-node politics implementation. General idea is that archived cache should stay "frozen" forever :) but I think it should be configurable

**Comment by rapotek** [comment]
> This is optional feature which allow per-node politics implementation. General idea is that archived cache should stay "frozen" forever :) but I think it should be configurable

OK, but the question is: if someone found a cache BEFORE it was archived but wanted to log id AFTER is was archived using the real found date, should it be allowed? As I remember, the topic has been raised several times on the forum.

If I understood you correctly, the policy for defined (configurable) subset of cache types should give a (configurable) choice between:

- allow logging all founds,
- prevent from logging if a found date was after an archived date,
- prevent from logging found at all if a cache is archived, regardless of a found date

Am I right?

**Comment by kojoty** [comment]
yes, in my opinion this issue is to implement both 3 ways of dealing with
such log
